src/harness/reference_models/geo/census_tract.py

```diff
@@ -16,11 +16,14 @@
 """
```

Comment:
I would prefer if the census tract driver was only used for reading the final json database, and not with a bunch of utilities to process shapefile, etc...

I mean all the routines you have below which do this processing do not really need to be in this class, as they do not need common shared object or data of the class. The only thing they use is the `self._census_tract_dir`, which can be a parameter of the script.

Hence we keep the driver very focused and simple.

I give more info below.

Reply:
Fixed   as per suggestion.